.github/workflows/cmake-windows.yml

```
@@ -17,50 +17,79 @@ jobs:
+        cat ~/.condarc
+    - name: Show current conda channels
+      run: conda config --show channels
+    - name: Enable long paths
+      run: |
+        echo "::group::Enable long paths"
+        reg add "HKLM\SYSTEM\CurrentControlSet\Control\FileSystem" /v LongPathsEnabled /t REG_DWORD /d 1 /f
+        git config --system core.longpaths true
+        echo "::endgroup::"
     - name: Build
       shell: bash -l {0}
       working-directory: ${{runner.workspace}}
```

actionlint (1.7.4)

property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)